### PR TITLE
Wildfire: update missing data warning notification

### PR DESCRIPTION
**Pull request by clhenrick**
Updates the warning message in the WildfireChart component to display the percentage of grid cells that have no data. 

Example:
Screen Shot 2021-11-17 at 2 04 53 PM

### CODE REVIEW

File: src/routes/tools/wildfire/_helpers.js [hunk 1]
```diff
@@ -22,3 +23,9 @@ export const getMapImages = ({
       palette,
     })
   );
+
+export const getAvgPctNoData = (dataStore) =>
```
Comment (clhenrick) [inline src/routes/tools/wildfire/_helpers.js:27]: Using mean here because each model and timestamp has its own `pctnd` value for no data.

**Comment by clhenrick** [comment]
[Update Wildfire API response to include a flag for no data values](https://trello.com/c/05YqnMvn/109-update-wildfire-api-response-to-include-a-flag-for-no-data-values)

File: src/routes/tools/wildfire/_constants.js [hunk 2]
```diff
@@ -40,19 +40,19 @@ represent a time series of annual/monthly simulations of average Area
 Burned/Decadal Wildfire Probability from individual downscaled GCMs.</p>
 <p>Click on any of the legend keys to highlight corresponding time series.</p>`;
 
+export const DATA_DISCLAIMER = `Locations outside the combined state and federal 
+fire protection responsibility areas were excluded from these wildfire 
+simulations and have no wildfire projections.`;
+
 export const MISSING_DATA_MSG = `This area may contain locations outside the 
 combined fire state and federal protection responsibility areas.
```
Comment (mukhtyar) [inline src/routes/tools/wildfire/_constants.js]: update text in `MISSING_DATA_MSG` to "combined state and federal fire protection responsibility areas"

**Review by mukhtyar — APPROVED** [review]

File: src/routes/tools/wildfire/_constants.js (hunk 2 above)
Comment (clhenrick) [reply src/routes/tools/wildfire/_constants.js]: This const is actually no longer being used so I'll remove it.

**Pull request merged**